tests/system/providers/google/cloud/kubernetes_engine/example_kubernetes_engine.py

```diff
@@ -44,7 +44,7 @@
 CLUSTER_NAME = CLUSTER_NAME_BASE if len(CLUSTER_NAME_FULL) >= 33 else CLUSTER_NAME_FULL
 
 # [START howto_operator_gcp_gke_create_cluster_definition]
-CLUSTER = {"name": CLUSTER_NAME, "initial_node_count": 1, "autopilot": {"enabled": True}}
+CLUSTER = {"name": CLUSTER_NAME, "node_pools": [{"initial_node_count": 1}], "autopilot": {"enabled": True}}
```

Comment:
@topherinternational did you try to run this DAG on your Google Cloud environment?
Because I got this error when I tried to run this DAG:
```
[2024-08-21T12:13:25.226+0000] {taskinstance.py:3296} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/google/api_core/grpc_helpers.py", line 76, in error_remapped_callable
    return callable_(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/grpc/_channel.py", line 1181, in __call__
    return _end_unary_response_blocking(state, call, False, None)
  File "/usr/local/lib/python3.8/site-packages/grpc/_channel.py", line 1006, in _end_unary_response_blocking
    raise _InactiveRpcError(state)  # pytype: disable=not-instantiable
grpc._channel._InactiveRpcError: <_InactiveRpcError of RPC that terminated with:
	status = StatusCode.INVALID_ARGUMENT
	details = "Node_pool.name must be specified."
	debug_error_string = "UNKNOWN:Error received from peer ipv4:66.102.1.95:443 {created_time:"2024-08-21T12:13:25.224438184+00:00", grpc_status:3, grpc_message:"Node_pool.name must be specified."}"
>
``` 
in my opinion it means that this `Cluster` configuration is incorrect